test/test_utils/security.py

```diff
@@ -277,21 +521,25 @@ def _save_lists_in_s3(save_details, s3_bucket_name):
         s3_client.upload_file(Filename=filename, Bucket=s3_bucket_name, Key=filename)
 
 
-def get_new_image_uri_for_uploading_upgraded_image_to_ecr(image):
+def get_new_image_uri_using_current_uri_and_new_repo(image, new_repository_name, new_repository_region, append_tag=""):
```

Comment:
nit: I'd recommend changing the names of the variables for repo and region to `target_repository_name` and `target_repository_region`, rather than `new_*`.

Reply:
Done